server/src/services/cleanUserInput.js

```diff
@@ -10,10 +10,19 @@ const cleanUserInput = formInput => {
     }
 
     if(field === "showId") {
-        const validId = Number(formInput.showId)
-        formInput.showId = validId
+        const validShowId = Number(formInput.showId)
+        formInput.showId = validShowId
     }
 
+    if(field === "userId") {
+        const validUserId = Number(formInput.userId)
+        formInput.userId = validUserId
+    }
+
+    if(field === "reviewId") {
+        const validReviewId = Number(formInput.reviewId)
+        formInput.reviewId = validReviewId
+    }
   })
```

Comment:
I strongly suggest refactoring this `cleanUserInput` to be dryer. Right now we do the same thing in 3 different places for ensuring an id is a number. Instead, you could designate an array of `numberProperties`, iterate over each and run this same check mid-iteration.